apps/cancel-request/fields/index.js

```diff
@@ -131,5 +141,23 @@ module.exports = {
     legend: {
       className: 'govuk-label--m'
     }
+  },
+  'cnc-email': {
+    mixin: 'input-text',
+    validate: [
+      'required',
+      { type: 'minlength', arguments: 6 },
+      { type: 'maxlength', arguments: 256 },
+      'email'
+    ]
+  },
+  'cnc-telephone': {
```

Comment:
I think this className: ['govuk-input', 'govuk-!-width-one-half'] has to be added for this field to match figma

Reply:
Good catch. I've updated the class now.